src/commands/run-tests.yml

```diff
@@ -23,6 +23,19 @@ parameters:
       semicolon-separated list.
     type: string
     default: ''
+  select-by-folder:
+    description: >
+      Location of the folder containing tests, relative to the project root folder.
+      To generate a test suite, MATLAB uses only the tests in the specified folder and its subfolders.
+      You can specify multiple folders using a colon-separated or a semicolon-separated list.
+    type: string
+    default: ''
+  select-by-tag:
+    description: >
+      Test tag used to select test suite elements. To generate a test suite, MATLAB uses only the test elements
+      with the specified tag.
+    type: string
+    default: ''
```

Comment:
@mw-hrastega does it seem a bit odd we document these two "select-by" parameters in a different way? It seems like maybe there should be a common wording for all "select-by" parameters. Perhaps:

"Folder used to select test suite elements. To generate a ..."
"Test tag used to select test suite elements. To generate a..."

Reply:
I see your argument, Mark. Yes, it is a good idea to follow the same pattern for arguments belonging to the same category. However, we might want to mention that select-by-folder contains a path. Maybe we can say: 

Location of the folder used to select test suite elements, relative to the project root folder. To generate...

What do you think?

Reply:
I agree to @mw-hrastega  updated accordingly.